app/Nova/Metrics/RsvpSourceBreakdown.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Nova\Metrics;
+
+use App\Rsvp;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Partition;
+
+class RsvpSourceBreakdown extends Partition
```

Comment:
How is this going to behave with a _lot_ of sources? For GI emails each visit token is used as a source.

Reply:
9067139 combines all the recruiting visit tokens into one item, and historical data says that we won't have enough to become a problem. Even if it does, the graph runs out of colors but the list scrolls and remains accurate.